.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,55 @@
+name: Bug report 🐛
```

> wiki나 pr을 한글로 적다보니 issue도 같은 글자로 통일해야한다 생각해서 우선 한글로 적어놨습니다.

PR 설명으로 봐서는 의도히시지 않으신 거죠? 이 템플릿의 이름도 아래 "기능 제안" 템플릿처럼 한글로 통일하는 게 나을 것 같아요.

```suggestion
name: 결함 제보 🐛
```

---

넵 이 부분은 의도는 없었어요. 한글로 수정하겠습니다!